engine/src/available-command.ts

```diff
@@ -131,14 +131,37 @@ export function generate(engine: Engine, subPhase: SubPhase = null, data?: any):
   return [];
 }
 
-function addPossibleNewPlanet(data: PlayerData, map: SpaceMap, hex: GaiaHex, pl: PlayerObject, planet: Planet,
-                              building: Building, buildings: any[]) {
+export function qicForDistance(distance: number, data: PlayerData): [number, boolean] {
```

Comment:
with typescript 4.x, you can name array arguments:

`export function(...): [qic: number, rangeBoosterUsed: boolean]` 

No runtime impact but it's useful to better understand the code